blocklylib-core/src/main/java/com/google/blockly/model/mutator/IfElseMutator.java

```diff
@@ -42,6 +46,27 @@ public IfElseMutator newMutator(BlocklyController controller) {
         }
     };
 
+    /**
+     * Writes an XML mutation string for the provided values.
+     *
+     * @param elseIfCount The count of {@code else if <test> <statement>} sections.
+     * @param hasElseStatement Whether the mutation should include a separate else statement.
+     * @return Serialized XML {@code <mutation>} tag, encoding the values.
+     */
+    public static String writeMutationString(
```

Comment:
Thought about making it private, but also thought someone out there might have need to make a similar ChangeEvent for some event stream. There isn't a good way to programmatically make the right ChangeEvent without this.